program-runtime/src/compute_budget.rs

```diff
@@ -276,6 +282,15 @@ impl ComputeBudget {
             .map(|fee_type| PrioritizationFeeDetails::new(fee_type, self.compute_unit_limit))
             .unwrap_or_default())
     }
+
+    // https://github.com/solana-labs/solana/issues/31453 to round compute-unit-price, in micro-lamports,
+    // down to its nearest 1_000 micro-lamport.
+    const ROUND_DOWN_FACTOR: u64 = 1_000;
```

Comment:
might have slightly better naming of this constant.

wdyt of `PRIORITY_FEE_TICK_SIZE`?

potentially even `PRIORITY_FEE_TICK_SIZE_MICROLAMPORTS` to clarify the units...

Reply:
Taking `PRIORITY_FEE_TICK_SIZE` but not biting `MICROLAMPORTS` bait 😸